docs/version0.46.1.md

```diff
@@ -0,0 +1,47 @@
+<!--
+* Copyright (c) 2017, 2024 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# What's new in version 0.46.1
+
+The following new features and notable changes since version 0.46.0 are included in this release:
+
+- [New binaries and changes to supported environments](#binaries-and-supported-environments)
+
+
+## Features and changes
+
+### Binaries and supported environments
+
+Eclipse OpenJ9&trade; release 0.46.1 supports OpenJDK 8, 11, 17, 21, and 22.
+
+Support for running OpenJ9 with all versions of OpenJDK on CentOS 6, CentOS 7, Red Hat Enterprise Linux (RHEL) 6, and RHEL 7 is removed.
+
+RHEL 8.6 and 9.0 are out of support. RHEL 8.8 is the new minimum operating system level.
```

Comment:
It should be RHEL 8.8 and 9.2.

Reply:
If RHEL 8.8 and 9.2 are now out of support then which is the new minimum operating system level? 

Also, do when then have to change the Supported environments topic? https://eclipse.dev/openj9/docs/openj9_support/

Reply:
The correction is:
RHEL 8.6 and 9.0 are out of support. RHEL 8.8 **and 9.2** are the new minimum operating system levels.

Like the previous comment, this isn't a change from 0.46.0. You could make this fix in that topic and remove the line here.